tests/tests-downloads.php

```diff
@@ -150,8 +108,8 @@ public function test_edd_download() {
 		$this->assertNotEmpty( $download3->prices );
 		$this->assertEquals( $prices, $download3->prices );
 		$this->assertEquals( $prices, $download3->get_prices() );
-		$this->assertEquals( 59, $download3->sales );
-		$this->assertEquals( 129.43, $download3->earnings );
```

Comment:
Actually using a decimal value here is testing that our rounding is working correctly. It needs to remain that one of these tests uses a decimal value so we continue to test that we aren't breaking rounding.